projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -105,34 +105,32 @@ function process_anchor_params() {
 		if ( ! \is_wp_error( $rss ) ) {
 			update_post_meta( $post->ID, 'jetpack_anchor_podcast', $podcast_id );
 
-			if ( ! empty( $episode_id ) ) {
```

Comment:
Why not add logic outside this conditional for when there isn't an episode id that get's the most recent episode using the helper?

```php
if ( empty( $episode_id ) ) {
    ...
```